ubuntu.18.04/README.md

```diff
@@ -1,5 +1,8 @@
 # Ubuntu WorkerTools
 
+> Please note that we update this document periodically to match the latest version on DockerHub which is publicly available.
+> This does not necessarily match the content of Dockerfiles in this repository, as they may contain changes that are not released yet.
+

```

Comment:
We should update the documentation after the versions are released to the public, because this readme is linked from our DockerHub page. So these words are added to avoid confusion.